infrastructure/database.tf

```diff
@@ -41,3 +42,48 @@ resource "azurerm_key_vault_secret" "POSTGRES-HOST" {
   value        = module.postgresql.fqdn
   key_vault_id = module.civil_sdt_key_vault.key_vault_id
 }
+
+
+module "postgresql-v15" {
+  source = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
+
+  providers = {
+    azurerm.postgres_network = azurerm.private_endpoint
+  }
+
+  admin_user_object_id = var.jenkins_AAD_objectId
+  business_area        = "cft"
+  common_tags          = local.tags
+  component             = var.component
+  env                  = var.env
+  pgsql_databases = [
+    {
+      name = "civil_sdt"
+    }
+  ]
+  pgsql_version        = "15"
+  product              = var.product
+  name               = join("-", [var.product, var.component, "v15"])
+  backup_retention_days = "${var.database_backup_retention_days}"
```

Comment:
Could you line up the equals signs with the backup_retention_days one?  The equals signs in the pgsql_databases section can be left as it is.

Reply:
Change has been made.